TappNetwork/home-visiting-app-web
# Android build

# Details
Changes for the Android build. 

# Steps to test
1. Run 'php artisan serve' in backend dir 
2. Change VUE_APP_ROOT_API in front end .env to "http://10.0.2.2:port of backend from artisan serve"
3. Open /android directory in android studio
4. Click sync project with gradle files button in upper right corner of android studio
5. Click run app button in upper toolbar in android studio
6. App should launch
7. If 'sign in' button does not function, open dev tools in chrome browser at 'chrome://inspect/' in the address bar, select he emulated device, and refresh after opening.
8. Navigate app and look for strange behavior/rendering

# Tasks Linked
[2m9dq2z](https://app.clickup.com/t/2m9dq2z)
